plugins/legacy-plugin-chart-table/src/controlPanel.tsx

```diff
@@ -22,41 +22,160 @@ import { t } from '@superset-ui/translation';
 import {
   formatSelectOptions,
   D3_TIME_FORMAT_OPTIONS,
+  ControlConfig,
   ColumnOption,
+  ControlStateMapping,
   ControlPanelConfig,
+  ControlPanelsContainerProps,
 } from '@superset-ui/chart-controls';
 import { validateNonEmpty } from '@superset-ui/validator';
 
+export enum QueryMode {
+  agg = 'agg',
```

Comment:
Perhaps full word `AGGREGATE` for clarity

Reply:
done